aweadmin.py

```diff
@@ -53,6 +55,50 @@ def get_jobs(baseurl,token,status):
   dumpdata(data)
   return data['data']
 
+def get_queues(baseurl,token,status):
+# need active here?
+  url = baseurl+"queue"+status #in-progress"
+  header={'Authorization': 'OAuth '+token}
+  req = requests.get(url, headers=header)
+
+  data = json.loads(req.text)
+
+  dumpdata(data)
+  return data['data']
+
+def move_to_penalty_box(baseurl, token, user, count, data, dryrun):
+  for d in data:
+    i= d['info']
+    id = d['id']
+    if i['user'] == user:
+      if i['clientgroups'] == "kb_upload_group":
+        args = [ id, "kb_upload_special" ]
+      else:
+        args = [ id, "special" ]
+      move_queued_job(baseurl,token,args,dryrun)
+
+def cap_jobs(baseurl,token, dryrun):
+  userCount = {}
+  #print("limit: %s" % cap_limit)
+  for status in ['in-progress','queued']:
+    data=get_jobs(baseurl,token,status)
+    for d in data:
+      i=d['info']
+      if not userCount.has_key(i['user']):
+        userCount[i['user']] = 0
+      userCount[i['user']] = userCount[i['user']] + 1
+
+  for user in userCount:
+    if userCount[user] > cap_limit:
+      move_to_penalty_box(baseurl, token, user, userCount[user], get_jobs(baseurl, token, "queued"), dryrun)
+
+def list_jobs(baseurl,token):
```

Comment:
Are there now two different list_jobs methods?  That seems wrong.  Can you consolidate into one function?